src/sentry/profiles/task.py

```diff
@@ -506,6 +506,7 @@ def on_symbolicator_request():
     if platform in SHOULD_SYMBOLICATE_JS:
         symbolicator_platform = SymbolicatorPlatform.js
     else:
+        # TODO @athena: remove comment before merging
         symbolicator_platform = SymbolicatorPlatform.native
```

Comment:
`symbolicator_platform` would be `SymbolicatorPlatform.native`  for anything none JS so this doesn't need to be changed.